### ISSUE TITLE
Improving dependencies

### ISSUE THREAD

**Issue by ReimarBauer** [Open-MSS/MSS#591]
**[Original report](https://bitbucket.org/wxmetvis/mss/issue/591) by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Looks like we still fixate too much

```
Package libtiff conflicts for:
basemap[version='>=1.2.2'] -> proj[version='>=7.2.0,<7.2.1.0a0'] -> libtiff[version='>=4.1.0,<5.0a0']
qt[version='>=5,<6'] -> gtk2 -> libtiff[version='>=4.0.10,<5.0a0|>=4.1.0,<5.0a0|>=4.0.9,<5.0a0|>=4.0.3,<4.0.8']
pillow[version='>=7.2.0'] -> libtiff[version='>=4.1.0,<5.0a0']

Package libpng conflicts for:
qt[version='>=5,<6'] -> libpng[version='>=1.6.32,<1.7.0a0|>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0']
basemap[version='>=1.2.2'] -> matplotlib-base[version='>=1.0.0,!=3.0.1'] -> libpng[version='>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0']
qt[version='>=5,<6'] -> fontconfig[version='>=2.13.0,<3.0a0'] -> libpng[version='>=1.6.28,<1.7|>=1.6.32,<1.6.35']
pillow[version='>=7.2.0'] -> freetype[version='>=2.10.4,<3.0a0'] -> libpng[version='>=1.6.34,<1.7.0a0|>=1.6.37,<1.7.0a0|>=1.6.35,<1.7.0a0']
pyqt[version='>=5,<6'] -> qt=5.9 -> libpng[version='>=1.6.32,<1.7.0a0|>=1.6.34,<1.7.0a0|>=1.6.35,<1.7.0a0|>=1.6.37,<1.7.0a0']

Package libgfortran4 conflicts for:
netcdf4 -> hdf5[version='>=1.10.6,<1.10.7.0a0'] -> libgfortran4[version='>=7.5.0']
scipy -> libgfortran4[version='>=7.5.0']
scipy -> libgfortran-ng[version='>=7,<8.0a0'] -> libgfortran4=7.5.0

Package libedit conflicts for:
qt[version='>=5,<6'] -> sqlite[version='>=3.25.3,<4.0a0'] -> libedit[version='>=3.1.20170329,<3.2.0a0|>=3.1.20181209,<3.2.0a0|>=3.1.20191231,<3.2.0a0']
python[version='>=3.7,<3.8.0a0'] -> sqlite[version='>=3.33.0,<4.0a0'] -> libedit[version='>=3.1.20170329,<3.2.0a0|>=3.1.20181209,<3.2.0a0|>=3.1.20191231,<3.2.0a0']
netcdf4 -> openssh -> libedit[version='>=3.1.20191231,<3.2.0a0']

Package pytz conflicts for:
fs_filepicker -> fs -> pytz
owslib -> pytz
pytz
skyfield[version='>=1.12'] -> pytz
basemap[version='>=1.2.2'] -> matplotlib-base[version='>=1.0.0,!=3.0.1'] -> pytz

Package krb5 conflicts for:
psycopg2 -> libpq[version='>=12.2,<13.0a0'] -> krb5[version='>=1.14.2,<1.15.0a0|>=1.16,<1.17.0a0|>=1.16.1,<1.17.0a0|>=1.16.4,<1.17.0a0|>=1.17.1,<1.18.0a0|>=1.16.3,<1.17.0a0|>=1.16.2,<1.17.0a0']
qt[version='>=5,<6'] -> libpq[version='>=12.3,<13.0a0'] -> krb5[version='>=1.17.1,<1.18.0a0']
git -> libcurl[version='>=7.65.3,<8.0a0'] -> krb5[version='1.14.*|>=1.14.6,<1.15.0a0|>=1.16.1,<1.17.0a0|>=1.17.1,<1.18.0a0|>=1.18.2,<1.19.0a0|>=1.16.4,<1.17.0a0|>=1.16.3,<1.17.0a0|>=1.16.2,<1.17.0a0']
netcdf4 -> openssh -> krb5[version='>=1.16.3,<1.17.0a0|>=1.17.1,<1.18.0a0']

Package pillow conflicts for:
matplotlib[version='>=3.3.0'] -> matplotlib-base[version='>=3.3.2,<3.3.3.0a0'] -> pillow[version='>=6.2.0']
basemap[version='>=1.2.2'] -> matplotlib-base[version='>=1.0.0,!=3.0.1'] -> pillow[version='>=6.2.0']
pillow[version='>=7.2.0']

Package cryptography-vectors conflicts for:
passlib -> cryptography -> cryptography-vectors[version='2.3.*|2.3.1.*']
pymysql[version='>=0.9.3'] -> cryptography -> cryptography-vectors[version='2.3.*|2.3.1.*']

Package proj4 conflicts for:
owslib -> pyproj[version='>=2'] -> proj4[version='6.0.0.*|>=6.0.0,<6.0.1.0a0|>=6.1.0,<6.1.1.0a0|>=6.1.1,<6.1.2.0a0|>=5.2.0,<5.2.1.0a0|>=5.1.0,<5.1.1.0a0|>=5.0.1,<5.0.2.0a0|>=5.2,<6|>=4.9.3,<4.9.4.0a0']
proj4[version='<6']
basemap[version='>=1.2.2'] -> pyproj -> proj4[version='6.0.0.*|>=5.0.1,<5.0.2.0a0|>=5.1.0,<5.1.1.0a0|>=5.2.0,<5.2.1.0a0|>=6.1.1,<6.1.2.0a0|>=6.1.0,<6.1.1.0a0|>=6.0.0,<6.0.1.0a0|>=5.2,<6|>=4.9.3,<4.9.4.0a0']

Package xorg-libx11 conflicts for:
qt[version='>=5,<6'] -> xorg-libx11
qt[version='>=5,<6'] -> xorg-libxext -> xorg-libx11=1.6

Package libxkbcommon conflicts for:
pyqt[version='>=5,<6'] -> qt[version='>=5.12.9,<5.13.0a0'] -> libxkbcommon[version='>=0.10.0,<1.0a0|>=0.9.1,<1.0a0|>=0.8.4,<1.0a0']
qt[version='>=5,<6'] -> libxkbcommon[version='>=0.10.0,<1.0a0|>=0.9.1,<1.0a0|>=0.8.4,<1.0a0']

Package certifi conflicts for:
netcdf4 -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
requests -> urllib3[version='>=1.21.1,<1.26,!=1.25.0,!=1.25.1'] -> certifi
fastkml -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
markdown[version='>=3.1.1'] -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
requests -> certifi[version='>=2017.4.17']
basemap[version='>=1.2.2'] -> matplotlib-base[version='>=1.0.0,!=3.0.1'] -> certifi[version='>=2020.06.20']
chameleon -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
pint -> setuptools -> certifi[version='>=2016.09|>=2016.9.26']
matplotlib[version='>=3.3.0'] -> matplotlib-base[version='>=3.3.2,<3.3.3.0a0'] -> certifi[version='>=2020.06.20']
owslib -> pyproj[version='>=2'] -> certifi[version='>=2017.4.17']

Package qt conflicts for:
matplotlib[version='>=3.3.0'] -> pyqt -> qt[version='4.8.*|5.6.*|5.9.*|>=5.9.7,<5.10.0a0|>=5.9.6,<5.10.0a0|>=5.9.4,<5.10.0a0|>=5.6.3,<5.7.0a0|>=5.12.9,<5.13.0a0|>=5.12.5,<5.13.0a0|>=5.6.2,<5.7.0a0|>=4.8.6,<5.0']
pyqt[version='>=5,<6'] -> qt[version='5.6.*|5.9.*|>=5.9.7,<5.10.0a0|>=5.9.6,<5.10.0a0|>=5.9.4,<5.10.0a0|>=5.6.3,<5.7.0a0|>=5.12.9,<5.13.0a0|>=5.12.5,<5.13.0a0|>=5.6.2,<5.7.0a0']
qt[version='>=5,<6']
fs_filepicker -> pyqt -> qt[version='4.8.*|5.6.*|5.9.*|>=5.9.7,<5.10.0a0|>=5.9.6,<5.10.0a0|>=5.9.4,<5.10.0a0|>=5.6.3,<5.7.0a0|>=5.12.9,<5.13.0a0|>=5.12.5,<5.13.0a0|>=5.6.2,<5.7.0a0|>=4.8.6,<5.0']

Package configparser conflicts for:
fs_filepicker -> fs.sshfs -> configparser
pint -> importlib_metadata -> configparser[version='>=3.5']
markdown[version='>=3.1.1'] -> importlib-metadata -> configparser[version='>=3.5']

Package curl conflicts for:
netcdf4 -> libnetcdf[version='>=4.7.3,<5.0a0'] -> curl[version='>=7.44.0,<8|>=7.59.0,<8.0a0|>=7.64.1,<8.0a0|>=7.69.1,<8.0a0|>=7.71.1,<8.0a0|>=7.55,<8|>=7.54.1,<8']
git -> curl[version='>=7.44.0,<8|>=7.59.0,<8.0a0|>=7.64.0,<8.0a0|>=7.64.1,<8.0a0|>=7.69.1,<8.0a0|>=7.71.1,<8.0a0']

Package libiconv conflicts for:
git -> libiconv[version='1.15|1.15.*|>=1.15,<1.17.0a0|>=1.16,<1.17.0a0']
lxml -> libxml2[version='>=2.9.10,<2.10.0a0'] -> libiconv[version='1.15|>=1.15,<1.17.0a0|>=1.16,<1.17.0a0']
qt[version='>=5,<6'] -> fontconfig[version='>=2.13.0,<3.0a0'] -> libiconv[version='1.15|>=1.15,<1.17.0a0|>=1.16,<1.17.0a0']

Package pcre conflicts for:
git -> pcre[version='>=8.41,<9.0a0|>=8.43,<9.0a0|>=8.44,<9.0a0']
qt[version='>=5,<6'] -> glib[version='>=2.56.2,<3.0a0'] -> pcre[version='>=8.41,<9.0a0|>=8.42,<9.0a0|>=8.43,<9.0a0|>=8.44,<9.0a0']

Package tornado conflicts for:
basemap[version='>=1.2.2'] -> matplotlib-base[version='>=1.0.0,!=3.0.1'] -> tornado
matplotlib[version='>=3.3.0'] -> matplotlib-base[version='>=3.3.2,<3.3.3.0a0'] -> tornado

Package importlib-metadata conflicts for:
markdown[version='>=3.1.1'] -> importlib-metadata
pint -> importlib_metadata -> importlib-metadata[version='>=1.1.3,<1.1.4.0a0|>=1.6.0,<1.6.1.0a0|>=1.6.1,<1.6.2.0a0|>=1.7.0,<1.7.1.0a0|>=2.0.0,<2.0.1.0a0|>=1.5.2,<1.5.3.0a0|>=1.5.0,<1.5.1.0a0']

Package typing conflicts for:
pint -> importlib_resources -> typing
fs_filepicker -> fs -> typing==3.6.4

Package pyqt conflicts for:
pyqt[version='>=5,<6']
matplotlib[version='>=3.3.0'] -> pyqt
fs_filepicker -> pyqt=5

Package pyproj conflicts for:
owslib -> pyproj[version='>=2']
basemap[version='>=1.2.2'] -> pyproj[version='>=1.9.3']

Package flask conflicts for:
flask-sqlalchemy -> flask[version='>=0.10|>=1.0.4']
flask
flask-socketio -> flask[version='>=0.9']
flask-httpauth -> flask

Package pygeoif conflicts for:
pygeoif
fastkml -> pygeoif

Package proj conflicts for:
basemap[version='>=1.2.2'] -> pyproj -> proj[version='>=6.2.0,<6.2.1.0a0|>=6.3.0,<6.3.1.0a0|>=6.3.1,<6.3.2.0a0|>=7.0.0,<7.0.1.0a0|>=7.0.1,<7.0.2.0a0']
basemap[version='>=1.2.2'] -> proj[version='>=6.2.1,<6.2.2.0a0|>=7.2.0,<7.2.1.0a0|>=7.1.1,<7.1.2.0a0|>=7.1.0,<7.1.1.0a0']

Package requests conflicts for:
requests
owslib -> requests[version='>=1.0']

Package cftime conflicts for:
netcdf4 -> cftime
cftime[version='>=1.0.1']

Package zstd conflicts for:
pillow[version='>=7.2.0'] -> libtiff[version='>=4.1.0,<5.0a0'] -> zstd[version='>=1.3.7,<1.3.8.0a0|>=1.4|>=1.4.4,<1.5.0a0|>=1.4.5,<1.5.0a0|>=1.4.4,<1.5.0.0a0|>=1.4.3,<1.5.0.0a0']
qt[version='>=5,<6'] -> mysql-libs[version='>=8.0.21,<8.0.22.0a0'] -> zstd[version='>=1.4.4,<1.5.0.0a0|>=1.4.5,<1.5.0a0']

Package xorg-libice conflicts for:
qt[version='>=5,<6'] -> xorg-libsm -> xorg-libice=1.0
qt[version='>=5,<6'] -> xorg-libice
```

‌

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

removed not needed fixations, fixes #591

**Comment by ReimarBauer** [comment]
**Original comment by Reimar Bauer (Bitbucket: [ReimarBauer](https://bitbucket.org/ReimarBauer), GitHub: [ReimarBauer](https://github.com/ReimarBauer)).**

----------------------------------------

Merged in ReimarBauer/mss (pull request #878)

removed not needed fixations, fixes #591
